tests/simulator/advance/test_fields_advance_2d.py

```diff
@@ -66,7 +66,7 @@ def test_field_coarsening_via_subcycles(self, interp_order, refinement_boxes):
 
     @data( # only supports a hierarchy with 2 levels
        *per_interp(({"L0": [Box2D(5, 9)]})),
-       *per_interp(({"L0": [Box2D(5, 24)]})),
+       *per_interp(({"L0": [Box2D(5, 19), Box2D(20, 24)]})),
```

Comment:
found this a bit odd, previously, the refined box was larger than the "largest_patch_size", so something in that?

Reply:
TODO track with issue

Reply:
there is something here under "mpirun -n 2"

Reply:
some cases are failing so something to assess in a new issue maybe